orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py

```diff
@@ -457,11 +457,19 @@ def testLRSchedulerUpdateImpl(lr_scheduler, expected_values):
                         expected_values[step], rtol=rtol, err_msg="lr mismatch")
 
 
-@pytest.mark.parametrize("step_fn", [
-    ('train_step'),
-    ('eval_step')
+@pytest.mark.parametrize("step_fn, lr_scheduler, expected_values", [
```

Comment:
It is not clear what "expected_values" holds. I am assuming it is expect lr values. If that is the case, I suggest renaming it to "expected_lr_values" to let this clearer